src/webview-contract/webviewDefinitions/clusterProperties.ts

```diff
@@ -1,3 +1,4 @@
+import { KubernetesVersionListResult } from "@azure/arm-containerservice";
```

Comment:
I'm actually surprised this works... The `webview-contract` files are referenced by both the main extension project and the `webview-ui` project. The `webview-ui` project does not have any Azure SDK libraries as dependencies, so I'd expect the `import` to fail (maybe it works because it's just a type annotation?). Shall we see if we can come up with a plain object-based structure that doesn't refer to any Azure libraries?

Reply:
Added a commit to address this.